aria-practices.html

```diff
@@ -2625,7 +2625,7 @@ <h4>Keyboard Interaction</h4>
             For additional guidance, see <a href="#kbd_selection_follows_focus"></a>.
           </li>
           <li>
-            If the tabs in a tab list are arranged vertically:
+            If the tabs in a tab list are arranged vertically (the property <a href="#aria-orientation" class="property-reference">aria-orientation</a> is set to <code>vertical</code>):
```

Comment:
Why is the added text in parenthesis?

Reply:
That works too.

Reply:
Could you please push those changes to your branch?

Reply:
I think I just did.

Reply:
Excellent, thank you very much.